episodes/04-git-changes.md

```diff
@@ -0,0 +1,310 @@
+
+In the Git tab of the upper left panel, we can click on the `Diff` icon.
+This will show changes made to `index.qmd`
+
+![](fig/RStudio_git_first_diff.png){alt='RStudio screenshot showing changes made to index.qmd and that these are unstaged.'}
+
+Git has noticed that there are changes to `index.qmd`, and that these are currently unstaged.
+Currently, Git doesn't know that it needs to keep track of these changes.
+
+To do this, we can check the box in the Staged column for `index.qmd`
+
+![](fig/RStudio_git_first_stage.png){alt='RStudio screenshot showing staging of index.qmd and change of ? to A under status column.'}
+
+We can see that under status the orange `?` has changed to a green `A` and in the bottom section the radio button for `Staged` is now selected.
+
+Git now knows that it's supposed to keep track of `index.qmd`,
+but it hasn't recorded these changes as a commit yet.
+
+![](fig/RStudio_git_first_commit.png){alt='RStudio screenshot showing initial commit message for index.qmd.'}
+
+To get it to do that, we need to type a message in the `Commit message` box and click on `Commit`.
+
+On the completion of the `commit` the following message is displayed in the terminal:
+
+```output
+>>> /usr/bin/git commit -F /var/folders/1v/6jw6cbqn1sldcbfrc24vj5qm0000gn/T/RtmpRsFsk9/git-commit-message-617d1bbdcbbe.txt
+[main (root-commit) a639e37] Added about information for the web page.
+ 1 file changed, 13 insertions(+)
+ create mode 100644 index.qmd
+
+```
+
+When we run `commit`,
+Git takes everything we have told it to save by using `stage`
+and stores a copy permanently inside the special `.git` directory.
+
+This permanent copy is called a [commit](../learners/reference.md#commit)
+(or [revision](../learners/reference.md#revision)) and its short identifier is `a639e37`. Your commit may have another identifier.
+
+We use the commit message to record a short, descriptive, and specific comment that will help us remember later on what we did and why.
+
+[Good commit messages][commit-messages] start with a brief (\<50 characters) statement about the
+changes made in the commit. Generally, the message should complete the sentence "If applied, this commit will" <commit message here>.
+If you want to go into more detail, add a blank line between the summary line and your additional notes. Use this additional space to explain why you made changes and/or what their impact will be.
+
+We can select `Diff "index.qmd"` from the Git dropdowb menu to show us the current status of the file `index.qmd`.
+
+![](fig/rs_git_diff_index_qmd.png){alt='RStudio screenshot showing selection of Diff index.qmd on Git dropdown menu.'}
+
+As everything is up to date, a dialogue box appears informing us that there have been no changes to the file.
+
+![](fig/rs_git_diff_no_change.png){alt='RStudio screenshot showing a dialogue box with the text "There are no changes to the file "index.qmd" to diff.".'}
```

Comment:
```suggestion
![](fig/rs_git_diff_no_change.png){alt='RStudio screenshot showing a dialogue box with the text "There are no changes to the file \"index.qmd\" to diff.".'}
```

I think... otherwise the first quotes will be finishing before index, and then start after the file name.